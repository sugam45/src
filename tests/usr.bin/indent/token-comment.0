/* $NetBSD: token-comment.0,v 1.2 2021/03/14 00:50:39 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Tests for formatting comments.  C99 defines block comments and end-of-line
 * comments.  Indent further distinguishes box comments that are a special
 * kind of block comments.
 */

/* TODO: Add some code to be formatted. */

/* 456789 123456789 123456789 123456789 123456789 123456789 123456789 12345 */
/* 456789 123456789 123456789 123456789 123456789 123456789 123456789 123456 */
/* 456789 123456789 123456789 123456789 123456789 123456789 123456789 1234567 */
/* 456789 123456789 123456789 123456789 123456789 123456789 123456789 12345678 */
/* 456789 123456789 123456789 123456789 123456789 123456789 123456789 123456789 */
