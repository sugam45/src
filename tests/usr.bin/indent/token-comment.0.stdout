/* $NetBSD: token-comment.0.stdout,v 1.3 2021/03/14 01:34:13 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Tests for formatting comments.  C99 defines block comments and end-of-line
 * comments.  Indent further distinguishes box comments that are a special
 * kind of block comments.
 */

/* TODO: Add some code to be formatted. */

/* 456789 123456789 123456789 123456789 123456789 123456789 123456789 12345 */
/* 456789 123456789 123456789 123456789 123456789 123456789 123456789 123456 */
/* 456789 123456789 123456789 123456789 123456789 123456789 123456789 1234567 */
/*
 * 456789 123456789 123456789 123456789 123456789 123456789 123456789 12345678
 */
/*
 * 456789 123456789 123456789 123456789 123456789 123456789 123456789
 * 123456789
 */
